### PR TITLE
creates default order status

### PR DESCRIPTION
**Pull request by tgraham777**
fixes photo thumbnail link on order display to link to the photo show page

### CODE REVIEW

File: app/controllers/orders_controller.rb [hunk 1]
```diff
@@ -30,6 +30,17 @@ def add_order_photos(order)
...
+  def add_default_order_status(order)
+    current_status = Status.find_by(name: "Ordered")
+    if current_status
+      order.status = current_status
+    else
+      new_status = Status.create(name: "Ordered")
+      order.status = new_status
+    end
+  end
+
```
Comment (mrjaimisra) [inline app/controllers/orders_controller.rb]: could this be refactored to "order.status = Status.find_or_create_by(name: "Ordered)"?

(thanks @sgrif !)

File: app/controllers/orders_controller.rb [hunk 2]
```diff
@@ -49,7 +60,8 @@ def add_order_photos(order)
   def create_order
     order = current_user.orders.new
     add_order_photos(order)
-    order.status_id = 1
+    add_default_order_status(order)
+    # order.status_id = 1
```
Comment (mrjaimisra) [inline app/controllers/orders_controller.rb]: nice

File: app/views/orders/show.html.erb [hunk 3]
```diff
@@ -25,7 +25,7 @@
                             width: 80,
                             class: "img"
                         ),
-                        photo_path(order_photo) %></td>
+                        photo_path(order_photo.photo_id) %></td>
```
Comment (mrjaimisra) [inline app/views/orders/show.html.erb]: why do we need to pass in the 'photo_id' here?

File: app/controllers/orders_controller.rb (hunk 1 above)
Comment (tgraham777) [inline app/controllers/orders_controller.rb]: Nice!

File: app/views/orders/show.html.erb (hunk 3 above)
Comment (tgraham777) [inline app/views/orders/show.html.erb]: the order_photo was not working as its own object, similar to the issue we had yesterday

**Comment by mrjaimisra** [comment]
awesome!

**Pull request merged**